### ISSUE TITLE
Commit event cell should be accessible

### ISSUE THREAD

**Issue by rnystrom** [rnystrom/GitHawk#1242]
Highlight the whole cell, not just the label

**Comment by BasThomas** [comment]
What class is this if I may ask? 😇

**Comment by rnystrom** [comment]
IssueCommitCell

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
